bot/cogs/animals.py

```diff
@@ -0,0 +1,127 @@
+"""Commands for the animal module."""
+import discord
+
+from discord.ext.commands import Cog, bot, command, context
+
+
+class AnimalCog(Cog, name='Animals'):
+    """Commands for the animal finder."""
+
+    def __init__(self, bot: bot.Bot):
+        self.bot = bot
+
+    @command(brief="Gets a random dog breed info and photo", aliases=['dogs'])
+    async def dog(self, ctx: context.Context) -> None:
+        """Gets random dog breed and facts about it."""
+        url = 'https://api.thedogapi.com/v1/images/search'
+
+        async with self.bot.session.get(url) as url:
```

Comment:
if we are using `Friendo` in the type hints, this problem will be fixed, but my IDE is trying to find a reference for `session` in the wrong place it appears.